README.md

```diff
@@ -196,39 +196,13 @@ This will return the full transcription of the video, transcription for each seg
 
 ### Running Example Applications
 
-Aana SDK comes with a set of example applications that demonstrate the capabilities of the SDK. You can run the example applications using the Aana CLI.
+We provide a few example applications that demonstrate the capabilities of Aana SDK.
 
-The following applications are available:
-- `chat_with_video`: A multimodal chat application that allows users to upload a video and ask questions about the video content based on the visual and audio information. See [Chat with Video Demo notebook](/notebooks/chat_with_video_demo.ipynb) for more information.
-- `whisper`: An application that demonstrates the Whisper model for automatic speech recognition (ASR).
-- `llama2`: An application that deploys LLaMa2 7B Chat model.
+- [Chat with Video](https://github.com/mobiusml/aana_chat_with_video): A multimodal chat application that allows users to upload a video and ask questions about the video content based on the visual and audio information. See [Chat with Video Demo notebook](https://github.com/mobiusml/aana_chat_with_video/blob/main/notebooks/chat_with_video_demo.ipynb) to see how to use the application.
+- [Summarize Video](https://github.com/mobiusml/aana_summarize_video): An Aana application that summarizes a video by extracting transcription from the audio and generating a summary using a Language Model (LLM). This application is a part of the [tutorial](https://mobiusml.github.io/aana_sdk/pages/tutorial/) on how to build multimodal applications with Aana SDK.
```

Comment:
reminder: The projects are still private.

Reply:
I will make them public once we merge the PRs.